Lab_13_Make_Money/script.js

```diff
@@ -0,0 +1,57 @@
+// get the form from HTML
+const moneyForm = document.getElementById("money-form")
+// get coin display div from HTML
+const coinDisplay = document.getElementById("coin-display")
+
+// add event listener so DOM knows what to do at submission
+moneyForm.addEventListener("submit", e => {
+    // prevent default submission behavior
+    e.preventDefault()
+    // at submission, get values for coin type and coin count from the form
+    const coinType = document.getElementById("coin-type").value
+    const coinCount = parseInt(document.getElementById("coin-count").value)
+
+    // testing out the button, not getting any results
+    /* 
+    const makeMoneyButton = document.querySelector("button")
+    makeMoneyButton.addEventListener("click", () => {
+        console.log(coinType)
+    }) 
+    */
+
+    // call functions below
+    hideCoinsAtStart()
+    displayCoins(coinType, coinCount)
+})
+
+// create function to hide coins at the start
+const hideCoinsAtStart = () => {
+    coinDisplay.style.display = "none"
+}
+
+// create function to display specified coins in their specified number
+const displayCoins = (coinType, coinCount) => {
+    coinDisplay.style.display = "inline-block"
+
+    const coinElements = document.querySelectorAll(`#coin-display .${coinType}-display`)
+
+    // loop to add the input number of coins to the page
+    for (let i = 0; i < coinCount; i++) { // for every coin leading up to input coin count
+        if (i < coinElements.length) { // as long as i < total coin count
+            coinElements[i].style.display = "inline-block" // it will keep displaying coins
+        }
+        // create event to hide the coin when it's clicked
+        coinElements[i].addEventListener("click", () => {
+        coinElements[i].style.display = "none"
+        })
+    } 
```

Comment:
```suggestion
        const coin = document.createElement('div')
        coin.classList.add(`${coinType}-display`) // string interpolation to match CSS class selector
        coin.innerText = coinType
        coin.addEventListener('click', () => coin.remove()) // remove from DOM instead of `display: none`
        coinDisplay.appendChild(coin)
    }
```